### DISCUSSION TITLE
Business Cooperation

### DISCUSSION THREAD

**Discussion by ViKingQ** [openTCS/opentcs#160] [General]
We are a company from China currently in need of utilizing this software to develop an automated scheduling plan for our clients. This involves integrating our scheduling system with AGVs (Automated Guided Vehicles), PLCs (Programmable Logic Controllers) and other devices. Our goal is to collaborate with openTCS, a widely-used industrial automation software, seeking support for integrating our systems. How do we proceed with this collaboration?

**Comment by swltr** [comment]
@ViKingQ Please see https://opentcs.org/en/contact.html for business contact information.
